projects/ngx-skeleton-loader/src/lib/ngx-skeleton-loader.component.spec.ts

```diff
@@ -190,3 +195,25 @@ describe('NgxSkeletonLoaderComponent', () => {
     });
   });
 });
+
+describe('NgxSkeletonLoaderComponent2', () => {
+  // tslint:disable-next-line: no-any
+  let fixture: any;
```

Comment:
nit: you could replace `any` here and elsewhere with `TestBedStatic` (which represents the return type of [**`TestBed.configureTestingModule`**](https://angular.io/api/core/testing/TestBedStatic#configureTestingModule)) to avoid besides the tslint disable comment, a possible typo or something else when using `fixture`.

```suggestion
  let fixture: TestBedStatic;
```

Reply:
I think this could be done in another PR in my opinion

Reply:
> I think this could be done in another PR in my opinion

My suggestion was focused in the additions, not to the existent code, however I don't see a problem on doing this on another PR too 😄 .

> EDIT: Also, I think fixture is of type `ComponentFixture<T>`

You're right. I was just looking for the `configureTestingModule(...)` part, missing the `compileComponent(...)`. Sorry for that.